### DISCUSSION TITLE
i am not able to host the file in github

### DISCUSSION THREAD

**Discussion by Harshit-Sevak** [community/community#132906] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Whenever I try to host it on github it is giving me this error 
![Screenshot (302)](https://github.com/user-attachments/assets/00ec8f92-2c5a-4f4b-a31a-615401030379)

**Comment by davevad93** [comment, marked as answer]
Hi @Harshit-Sevak ,

That usually happens with repos hosted on GitHub Pages cloning the design of popular websites like `Facebook`, `Amazon`, `Instagram`, `Twitter`, `Netflix`, `Spotify`. That's a `Google` security policy, for more information see the Google Safe Browsing documentation on [Social engineering (phishing and deceptive sites)](https://developers.google.com/search/docs/advanced/security/social-engineering?hl=en).   

There's also another discussion from a couple of months ago regarding this if you want further reference. #125656

**Comment by Harshit-Sevak** [comment]
But whichever file I am hosting, is showing the same error.

**Reply by davevad93** [reply]
That's true, I've checked your repos and all of them are marked as dangerous. As I said that usually happens when google detects that a page might be a phishing website (like a clone of another web). The issue might be related to Google Safe Browsing phishing list which is marking all your hosted projects. Take a look at this [comment](https://github.com/orgs/community/discussions/30261#discussioncomment-3467665) from an old discussion about the same subject, maybe you can find a solution there.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Harshit-Sevak ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
